### PR TITLE
Quttera integration

### PR DESCRIPTION
**Pull request by hayzer**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Review by edik24 — COMMENTED** [review]
the file
Packs/QutteraWebsiteMalwareScanner/Integrations/README.md
is not needed

File: Packs/QutteraWebsiteMalwareScanner/pack_metadata.json [hunk 1]
```diff
@@ -0,0 +1,36 @@
+{
+    "name": "QutteraWebsiteMalwareScanner",
+    "description": "Detect suspicious/malicious/blocklisted content on domains/URLs. Run real-time normal/heuristic scan and database queries.",
+    "support": "partner",
+    "currentVersion": "1.0.0",
+    "author": "Quttera LTD",
+    "url": "",
```
Comment (edik24) [inline Packs/QutteraWebsiteMalwareScanner/pack_metadata.json:7]: need to add url

Comment (hayzer) [reply Packs/QutteraWebsiteMalwareScanner/pack_metadata.json:7]: Fixed

**Comment by ShahafBenYakir** [comment]
What is the status here? @hayzer?

**Comment by edik24** [comment]
closed, they will open a new one when relevant

**Pull request closed without merging**